### PR TITLE
[DO NOT MERGE] add demo and ugly solution to solve duplicated files with same name

### PR DESCRIPTION
**Pull request by sihu**

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  $(document).ready(function () {
+    var dropzoneOptions = {
+      acceptedFiles: 'image/*,application/pdf',
+      dictDefaultMessage: 'Ziehen Sie eine Bild-Datei oder ein PDF hier herein, oder klicken Sie hier um die Datei hochzuladen.',
+      dictFallbackMessage: 'Sie verwenden einen veralteten Browser was das verwenden des Uploads verhindert.',
+      dictInvalidFileType: 'Hier können nur Bild-Dateien hochgeladen werden.',
+      dictFileTooBig: 'Die Datei ist zu gross.'
+    };
+    new window.RenuoUpload($('.renuo-upload'), dropzoneOptions, function (a, b, c) {
+      console.log(a);
```
Comment (cyrilkyburz) [inline demo/index.html:52]: nice callback

Comment (cyrilkyburz) [inline demo/index.html:52]: here would be another example: https://github.com/renuo/renuo-upload#complete-example

**Comment by cyrilkyburz** [comment]
What do we plan to do?

**Comment by sihu** [comment]
i think it has to be refactored anyways. so you can also take the ideas and write it from scratch?

**Comment by cyrilkyburz** [comment]
I will create a ticket for it.

**Comment by sihu** [comment]
so do we close the branch?

**Comment by cyrilkyburz** [comment]
I have the branch as a reference, I guess the ticket will delete the branch.

**Pull request closed without merging**